test/test_datasets.py

```diff
@@ -58,12 +58,12 @@ def test_openfire(self):
             self.assertTrue(all(sample['path'].name.rpartition('.')[-1] in ['jpg', 'jpeg', 'png', 'gif']
                                 for sample in test_set.data))
 
-            # Check against number of samples in extract
+            # Check against number of samples in extract (limit to 200)
```

Comment:
Small update to reflect my previous comment would be nice!

Reply:
Done